controller/templates/confd_settings.py

```diff
@@ -79,3 +79,12 @@
 PRIVATE_EXT_REGISTRY_FLAVOR = '{{ getv "/deis/controller/private_ext_registry/flavor" }}'
 PRIVATE_EXT_REGISTRY_DATA = json.loads('''{{ getv "/deis/controller/private_ext_registry/data" }}'''.replace("\'", '"'))
 {{ end }}
+
+# if defined, then scheduler will distribute the units cross that list.
+# it is a list of anything, the scheduler would pick az = AVAILABILITY_ZONE_LIST[num % len(AVAILABILITY_ZONE_LIST)]
+# which means if list is ['a', 'b'], and it is the 3rd container of the same application,
+# we would do, 3 % 2 = 1 => az = list(1) => az=b
+AVAILABILITY_ZONE_LIST=[]
```

Comment:
I don't think what you've posted is related to instantiating an empty list/dict. Line 8 of this file has a good example of what I'm recommending:

    try:
        SCHEDULER_OPTIONS = dict('{{ if exists "/deis/controller/schedulerOptions" }}{{ getv "/deis/controller/schedulerOptions" }}{{ else }}{}{{ end }}')
    except:
        SCHEDULER_OPTIONS = {}

Although if you want to use `if exists` instead you can just do it an `if...else`